### PR TITLE
website/docs: add example policy to enforce unique email address

### PR DESCRIPTION
**Pull request by BeryJu**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/policies/working_with_policies/unique_email.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Ensure unique email addresses
+---
+
+Due to the database design of authentik, Email addresses are by default not required to be unique. This behavior can however be changed by policies.
```
Comment (rissson) [inline website/docs/policies/working_with_policies/unique_email.md:5]:
```suggestion
Due to the database design of authentik, email addresses are by default not required to be unique. This behavior can however be changed by policies.
```

**Review by rissson — APPROVED** [review]

File: website/docs/policies/working_with_policies/unique_email.md (hunk 1 above)
Comment (tanberry) [inline website/docs/policies/working_with_policies/unique_email.md:5]:
```suggestion
Due to the database design of authentik, email addresses are by default not required to be unique. This behavior can however be changed by policies.
```

File: website/docs/policies/working_with_policies/unique_email.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+The policy snippet below can be used both with enrollment flows, where the policy should be bound to any stage before the [User write](../../flow/stages/user_write.md) stage, or it can be used with the [Prompt stage](../../flow/stages/prompt/index.md).
```
Comment (tanberry) [inline website/docs/policies/working_with_policies/unique_email.md:7]: Can we say where the user needs to add this snippet? Is it added in the Expression field of the policy? Or... ?

**Comment by tanberry** [comment]
Also, I am glad to see this documented, since people do notice and ask about it...

**Pull request merged**